finagle-thrift/src/main/scala/com/twitter/finagle/thrift/ThriftClientBufferedCodec.scala

```diff
@@ -28,6 +28,8 @@ object ThriftClientBufferedCodec {
    */
   def apply() = new ThriftClientBufferedCodecFactory
 
+  def get() = apply()
```

Comment:
can you add a return type here?  public methods should be annotated (it's OK not to fix up the rest of the file).